x-pack/solutions/observability/plugins/slo/public/pages/slo_details/components/slo_health_callout.tsx

```diff
@@ -49,64 +82,26 @@ export function SloHealthCallout({ slo }: { slo: SLOWithSummaryResponse }) {
           <FormattedMessage
             id="xpack.slo.sloDetails.healthCallout.description"
             defaultMessage="The following {count, plural, one {transform is} other {transforms are}
-          } in an unhealthy state:"
+          } in an unhealthy state. Data may be missing or incomplete. You can inspect {count, plural, it {one} other {each one}} here:"
```

Comment:
Do we need to remove the translations if we modify this message? So that they can get recreated.